packages/kotti-ui/source/kotti-field-select/types.ts

```diff
@@ -55,6 +55,7 @@ export namespace KottiFieldSingleSelect {
 export namespace KottiFieldSingleSelectRemote {
 	export type Props = KottiField.Props<Value, string | null> &
 		Shared.Props & {
+			dataTest: string | null
```

Comment:
I don't see this being used in the component anywhere.

Reply:
because the `modifiedOptions` that are passed, merely `...option` then add more attributes to it 

but you should see a change on the template level where I bound `dataTest` to `option.dataTest`

Reply:
so basically - what kind of change would be needed logically, even? it's just the prop passed down, then remapped by the `modifiedOptions` declaration

Reply:
Sure, but I would expect this `dataTest` on the field level, not on the option level. 
Like so:
```html
<KtField
	v-bind="{ field }"
	class="kt-field-select kt-field-select--single-remote"
	:data-test="dataTest"
	:getEmptyValue="() => null"
	:helpTextSlot="$slots.helpText"
>
```

Reply:
The dataTest that are passed to the options are completely generated and are not derived from the prop that is supposedly passed here.


...by the way, am I stupid or is `isLoadingOptions` also effectively doing nothing in the component?

Reply:
NVM - saw what you meant

Reply:
I misplaced the data-test in the type definition of the field itself, yes.

it belonged on the options, and I added it and its corresponding logic in later commits.

didn't check `isLoadingOptions` but can do

no the ones on the options are NOT generated automatically. On the contrary, the ones on the fields are AUTOMATICALLY generated from the `formId` and the `formKey` and the hierarchy
> so I removed the `dataTest` from the field-level (my mistake initially)
- if the user wants to tweak the data-test, they can tweak the `formId`
- but there is no allowance through the API to tweak it per field 
- because we always relied on `formPath`

This ^ is a limitation however, because not every field is used within a `KtForm`
so needs fixing in the future

Reply:
Got it